README.md

```diff
@@ -45,8 +45,17 @@ The app targets shoppers across Australia who are looking for unique and quality
 - Japanese food products lovers
 
 ## User Stories 
+We construct user stories and personas to improve the online store's user experience for Japanese food products. The process involves identifying and embodying potential users' unique needs and expectations, including a system administrator at Sakura company.
 
-Working in progress...
+A problem Shizuka faces is the difficulty travelling to supermarkets where authentic Japanese food products are available, even with her three children. With the demands of raisingtrello  her children, she also needs to save time when shopping for her daily food. She is particularly concerned about the security of websites, especially when shopping online, and prefers to use sites that offer a secure login facility.
+![Shizuka-Kagami](./docs/images/user-pesonas/Sizuka-kagami.png)
+
+On the other hand, Ronnie, a user who has recently become addicted to Japanese food, also faces difficulties in finding Japanese foodstuffs locally. Ronnie is also an engineer. The engineer, Ronnie, also expects the system's ease of use and accessibility to the information required.
```

Comment:
This sentence, you can add some more info:

`On the other hand, Ronnie, a user who has recently become addicted to Japanese food, also faces difficulties in finding Japanese foodstuffs locally`

--> "On the other hand, Ronnie, a user who has recently become addicted to Japanese food, also faces the difficulty of finding unique and authentic Japanese food products locally. He also prefer a clear product description written in English as he has a hard time choosing the right product when the product labels are mostly written in Japanese."